packs/st2/actions/kv_get.py

```diff
@@ -13,7 +13,7 @@ def run(self, key, decompress=False):
         _key = self.client.keys.get_by_name(key)
 
         if not _key:
-            return None
+            raise Exception("Key does not exist")
```

Comment:
Agreed. `None` is not a great return value. 

OTH a generic exception is only so much better. Since this will be in a workflow and the exact nature of the failure is probably not important generic exception should be fine.

Do watch out for regressions.